pkg/v1/tarball/write_test.go

```diff
@@ -373,3 +374,44 @@ func TestWriteSharedLayers(t *testing.T) {
 		}
 	}
 }
+
+func ExampleWriteToFileWithProgress() {
```

Comment:
I think this function has to be named exactly `ExampleWithProgress` in order to show up under `WithProgress` in godoc.

Reply:
I was wondering about that....

Reply:
Yup. Now the linter stopped complying.